shell/platform/linux/fl_view.cc

```diff
@@ -200,8 +200,11 @@ static void handle_geometry_changed(FlView* self) {
       self->engine, allocation.width * scale_factor,
       allocation.height * scale_factor, scale_factor);
 
-  fl_renderer_wait_for_frame(self->renderer, allocation.width * scale_factor,
-                             allocation.height * scale_factor);
+  if (allocation.width > 1 && allocation.height > 1 &&
```

Comment:
Why is the resolution check for > 1? This seems unexpected (would expect > 0), so probably needs a comment above explaining.

Reply:
Indeed, this deserves a clarifying comment. It's because `gtk_widget_init()` initializes it to 1x1.
```c
priv->allocation.width = 1;
priv->allocation.height = 1;
```

https://github.com/GNOME/gtk/blob/4e3a3f05533789e1a68c70c185e1755d386d6c47/gtk/gtkwidget.c#L4421-L4422